src/dbnode/client/write_state.go

```diff
@@ -37,6 +37,18 @@ import (
 	"go.uber.org/zap"
 )
 
+type countTowardsConsistency int64
+
+const (
+	undefined countTowardsConsistency = iota
```

Comment:
nit: prefix these names with the enum type -- instead of `undefined`, do `countTwoardsConsistencyUndefined`